daemon/api_users.go

```diff
@@ -269,15 +269,19 @@ func removeUser(c *Command, username string, opts postUserDeleteData) Response {
 
 	// then the UserState
 	st.Lock()
-	err = auth.RemoveUserByUsername(st, username)
+	u, err := auth.RemoveUserByUsername(st, username)
 	st.Unlock()
 	// ErrInvalidUser means "not found" in this case
 	if err != nil && err != auth.ErrInvalidUser {
 		return InternalError(err.Error())
 	}
 
-	// returns nil so it's still arguably a []userResponseData
-	return SyncResponse(nil, nil)
+	result := map[string]interface{}{
```

Comment:
Do we have to update any API documentation as a result of this change?

Reply:
There is no removeUser in 2.43 so this shoudl be fine.

Reply:
I mean, do we need to document the API as this is a official snapd API and those are usually documented.

Reply:
Yes but our API docs still live in the wiki and are not up-to-date (nothing about snapshots in them for example), is something we do need to fix, but this PR doesn't make that particularly worse or better.